plugins/org.eclipse.n4js/src/org/eclipse/n4js/xtext/workspace/WorkspaceConfigSnapshot.java

```diff
@@ -38,47 +38,55 @@
 	protected final ImmutableMap<URI, ProjectConfigSnapshot> projectPath2Project;
 	/** Keys are URIs <em>without</em> trailing path separator. */
 	protected final ImmutableMap<URI, ProjectConfigSnapshot> sourceFolderPath2Project;
+	/** Project build order */
+	protected final ProjectBuildOrderInfo projectBuildOrderInfo;
```

Comment:
One more thing: does this new field need to be considered in methods `#hashCode()` and `#equals()`? If not, add a comment in those methods that it does not have to be considered (as already the case for the look-up maps).